admin/includes/auto_loaders/config.core.php

```diff
@@ -110,6 +110,15 @@
   $autoLoadConfig[30][] = array('autoType'=>'classInstantiate',
                                 'className'=>'sniffer',
                                 'objectName'=>'sniffer');
+/**
+ * Breakpoint 32.
+ *
+ * $messageStack = new messageStack();
+ *
+ */
+  $autoLoadConfig[32][] = array('autoType'=>'classInstantiate',

```

Comment:
@mc12345678 any recollection about why you chose 32 here?

Reply:
Certainly.

First there was some level of evaluation about how early the `message_stack` should be made available.  In review of the code loaded in `sniffer` and before, it did not appear that there was much cause/need for the `message_stack` to yet be available. So it seemed as if any earlier than LP 30 wasn't necessary.  Then in review of the code that followed, there seemed like there may be a desire/opportunity to use the stack for subsequent files, there was an expressed need for the files at LP 35, so it seemed that somewhere between the two seemed reasonable.

Then there was the thought of how close to each should it be placed.  If I could have chosen 32.5 I would have used it, but that was not considered available.  I didn't see a need for it to be *immediately* after 30 and didn't see why it should be immediately before 35 nor 35 itself without also knowing how others might have loaded files after/before (respectfully) each of these two LPs.  So, I went with the slightly further away from 35 as an option.  It almost seems reasonable that the class could be loaded at the end of LP 0 and before LP 10, but I didn't see a specific need nor was I sure of potential plans of how to use it now that additional options have been made available.

So, in short it fit the described need, didn't seem to get in the way of anything else, but appears that it could be used for more of the load system than currently targeted.  The separation was already somewhat into the next direction and possibly hard to accept so didn't want to be too drastic with this one. :) It certainly seems that it could be loaded sooner.

Reply:
Thanks.